src/utils/filter-utils.js

```diff
@@ -489,6 +513,20 @@ export function isInRange(val, domain) {
   return val >= domain[0] && val <= domain[1];
 }
 
+/**
+ * Determines whether a point is within the provided polygon
+ *
+ * @param point as input search [lat, lng]
+ * @param polygon Points must be within these (Multi)Polygon(s)
+ * @return {boolean}
+ */
+export function isInPolygon(point, polygon) {
+  const convertedPoint = turfPoint(point);
+  const convertedPolygon = turfPolygon(polygon.geometry.coordinates);
```

Comment:
Here you convert filter.value every time. should only need to do it once. you can store convertedPolygon in polygon filter?

Reply:
wouldn't it be more performant to http://turfjs.org/docs/#booleanPointInPolygon?

Reply:
> wouldn't it be more performant to http://turfjs.org/docs/#booleanPointInPolygon?

I am not sure booleanPointInPolygon is more performant. When we convert a point to a turf point, the underline representation of convertedPoint is a geojson object hence i am not sure about performance.
On the other hand booleanWithin is more flexible and we can pass any kind of shape (not just points) through geojson therefore we could use for different layers